packages/kbn-config-schema/src/types/duration_type.test.ts

```diff
@@ -51,6 +51,14 @@ describe('#defaultValue', () => {
     ).toMatchSnapshot();
   });
 
+  test('can be a string-formatted number', () => {
```

Comment:
__nit:__ can you please also add a happy-path test for the case when string is provided as a value, not only as a `defaultValue` (outside of `'#defaultValue'` test suite)? Same for the `byteSize` test above.